src/colmap/geometry/rigid3_test.cc

```diff
@@ -83,6 +83,12 @@ TEST(Rigid3d, Inverse) {
   }
 }
 
+TEST(Rigid3d, TgtOriginInSrc) {
+  const Rigid3d b_from_a = TestRigid3d();
+  const Eigen::Vector3d origin_in_b = TgtOriginInSrc(b_from_a);
+  EXPECT_LT((b_from_a * origin_in_b - Eigen::Vector3d::Zero()).norm(), 1e-6);
```

Comment:
Shouldn't this be origin_in_a?
```suggestion
  const Eigen::Vector3d origin_in_a = TgtOriginInSrc(b_from_a);
  EXPECT_LT((b_from_a * origin_in_a - Eigen::Vector3d::Zero()).norm(), 1e-6);
```